test/jdk/java/util/Formatter/Basic-X.java.template

```diff
@@ -1745,5 +1745,11 @@ public class Basic$Type$ extends Basic {
         tryCatch("%%%", UnknownFormatConversionException.class);
         // perhaps an IllegalFormatArgumentIndexException should be defined?
         tryCatch("%<%", IllegalFormatFlagsException.class);
+
```

Comment:
After adding test-cases to this template you need to run `genBasic.sh` (in the same folder) to regenerate all the `Basic*` tests. These changes then need to be added to the PR.

FWIW these new and existing tests that provoke `IllegalFormatPrecisionException` and `UnknownFormatConversionException` are common to all types, meaning there seems to be little point in generating them out into each and every `BasicByte`, `BasicInt` et.c.. Perhaps it would make more sense to add such tests directly to Basic.java to avoid some redundancy here - or a new `BasicCommon` test to put common tests into?